phpstan-baseline.neon

```diff
@@ -1,2 +1,11 @@
 parameters:
-	ignoreErrors:
\ No newline at end of file
+	ignoreErrors:
+		-
+			message: "#^Call to method addSetting\\(\\) on an unknown class Pimcore\\\\Bundle\\\\AdminBundle\\\\Event\\\\IndexActionSettingsEvent\\.$#"
```

Comment:
Maybe we can even add this listeners to a separate file like here:
https://github.com/pimcore/pimcore/blob/11.x/bundles/StaticRoutesBundle/config/admin-classic.yaml
We then also only load them when the classic bundle is installed:
https://github.com/pimcore/pimcore/blob/11.x/bundles/StaticRoutesBundle/src/DependencyInjection/PimcoreStaticRoutesExtension.php#L49